packages/core/src/domain/internalMonitoring/internalMonitoring.ts

```diff
@@ -71,15 +71,17 @@ export function startInternalMonitoring(configuration: Configuration): InternalM
   function toTelemetryEvent(message: MonitoringMessage): TelemetryEvent & Context {
     return combine(
       {
+        type: 'telemetry' as const,
         date: timeStampNow(),
         service: 'browser-sdk',
         version: __BUILD_ENV__SDK_VERSION__,
+        source: 'browser' as const,
         _dd: {
-          event_type: 'internal_telemetry' as const,
+          format_version: 2 as const,
         },
       },
       telemetryContextProvider !== undefined ? telemetryContextProvider() : {},
-      message
+      { telemetry: message as any } // https://github.com/microsoft/TypeScript/issues/48457
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏move the `telemetry` property in the object passed as first argument?

Reply:
Yes indeed